### DISCUSSION TITLE
Customize (Side bar menu)

### DISCUSSION THREAD

**Discussion by johlems** [snipe/snipe-it#11879] [Q&A]
Hi everyone!

May i ask if it's possible to customize the sidebar on SnipeIT?

For example if I wanted to change the label **"All Deployed"** to **"All Devices"**

Attachment for reference. Thank you for responding.

Regards,
![Label](https://user-images.githubusercontent.com/19777107/192538168-dd1e8ea0-2c2e-4f8b-9f26-0b875330d25f.png)

**Comment by rasari03** [comment]
Is it possible to stop the Side bar from scrolling with the main page. If I scroll down and I need to access side bar , I have to scroll all the way back to top of page.
